### PR TITLE
Add: AWX job action

### PR DESCRIPTION
**Pull request by pascalholthaus**
## What
Add: AWX job action

## Why
This enables us to initiate jobs and workflows on an AWX server, as well as update a project within AWX.

## References
DEVOPS-978

## Checklist
Tested in pipeline-experiments.

### CODE REVIEW

File: awx-run/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+```yaml
+jobs:
+  awx-run:
+    name: Start a AWX job and or project update
+    runs-on:
+      - self-hosted
+      - self-hosted-generic
+    steps:
+      - name: Start and wait for a AWX job template
+        uses: ./.github/actions/awx-run
```
Comment (bjoernricks) [inline awx-run/README.md:16]:
```suggestion
        uses: greenbone/actions/awx-run@v3
```

File: awx-run/action.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ inputs.python-version }}
+
+    - name: Install Ansible
+      if: ${{ !startsWith(runner.name, inputs.skip-installation-on) }}
+      shell: bash
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install --upgrade ansible
```
Comment (bjoernricks) [inline awx-run/action.yml:71]: would be better to install it with pipx

Comment (pascalholthaus) [reply awx-run/action.yml:71]: I have to try it last time i used pipx it didn't link all binary's into /bin.

Comment (pascalholthaus) [reply awx-run/action.yml]: Changed

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**